app/models/patient.rb

```diff
@@ -227,11 +233,21 @@ def latest_transfer
     where(monitoring_reason: 'Case confirmed')
   }
 
+  # Any individual needing a follow up 
+  scope :followup, lambda {
+    where(monitoring: true)
+      .where(purged: false)
+      .where(public_health_action: 'None')
+      .where.not(severe_symptom_onset: nil)
+      .distinct
+  }
+
   # Any individual who has any assessments still considered symptomatic (includes patients in both exposure & isolation workflows)
   scope :symptomatic, lambda {
     where(monitoring: true)
       .where(purged: false)
       .where(public_health_action: 'None')
+      .where(severe_symptom_onset: nil)
       .where.not(symptom_onset: nil)
       .distinct
   }
```

Comment:
@ChristineDuong @kevstoy Not saying this is wrong or doesnt make sense, but just want to verify that the logic here that a patient will never appear in BOTH the symptomatic and severe symptomatic columns.